docs/intro.rst

```diff
@@ -363,7 +363,7 @@ The two tabs below provide the two formulations.
         capacity.
 
         >>> total_weight = items * weight 
-        >>> model.add_constraint(total_weight.sum() <= capacity)
+        >>> _ = model.add_constraint(total_weight.sum() <= capacity)
```

Comment:
I am so annoyed by Python that [we cannot just do](https://bugs.python.org/issue32509)
```python
>>> model.add_constraint(total_weight.sum() <= capacity)  # doctest: +ELLIPSIS
...
```

IMO
```python
>>> model.add_constraint(total_weight.sum() <= capacity)  # doctest: +ELLIPSIS
<dwave.optimization.symbols.LessEqual at ...>
```
is clearer.

@JoelPasvolsky, any opinions?

Reply:
I'll update this in the docs

Reply:
Here and elsewhere please.

Reply:
@arcondello updated